From d7baa6a9163bfabea2cffc42e14d7f045ab2d188 Mon Sep 17 00:00:00 2001
From: Christian Rauch <Rauch.Christian@gmx.de>
Date: Thu, 23 May 2024 21:10:53 +0200
Subject: [PATCH] re-sign IPA plugins

---
 rpm/template.spec.em | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index cc2626a..988cfac 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -49,6 +49,13 @@ if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.
     %{_target_platform}
 %meson_build -C %{_target_platform}
 
+%define __spec_install_post \
+    %{?__debug_package:%{__debug_install_post}} \
+    %{__arch_install_post} \
+    %{__os_install_post} \
+    %{_builddir}/src/ipa/ipa-sign-install.sh %{_builddir}/%{name}-%{version}/%{_vpath_builddir}/src/ipa-priv-key.pem %{buildroot}/%{_libdir}/libcamera/ipa_*.so \
+%{nil}
+
 %install
 # In case we're installing to a non-standard location, look for a setup.sh
 # in the install tree and source it.  It will set things like
-- 
2.43.0

