From 3389bca4610edd17cf79f1178cbedf54932e1657 Mon Sep 17 00:00:00 2001
From: Christian Rauch <Rauch.Christian@gmx.de>
Date: Tue, 8 Apr 2025 23:04:12 +0200
Subject: [PATCH 2/3] allow downloading subproject "libpisp"

---
 debian/rules.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index 6c7d27e0fcd2..c839a96fe7be 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -29,6 +29,8 @@ DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 override_dh_auto_configure:
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
+		--wrap-mode=forcefallback \
+		--force-fallback-for=libpisp \
 		--prefix="@(InstallationPrefix)" \
 		--cmake-prefix-path="@(InstallationPrefix)" \
 		--libdir=lib \
-- 
2.49.0

