From bd271bcff49d227b7e213d3d6d93379e89f5de3a Mon Sep 17 00:00:00 2001
From: Christian Rauch <Rauch.Christian@gmx.de>
Date: Sun, 1 Sep 2024 20:21:28 +0200
Subject: [PATCH 2/2] fix IPA signing

---
 rpm/template.spec.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index 742e389..3db5212 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -53,7 +53,7 @@ if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.
     %{?__debug_package:%{__debug_install_post}} \
     %{__arch_install_post} \
     %{__os_install_post} \
-    %{_builddir}/src/ipa/ipa-sign-install.sh %{_builddir}/%{name}-%{version}/%{_vpath_builddir}/src/ipa-priv-key.pem %{buildroot}/%{_libdir}/libcamera/ipa_*.so \
+    %{_builddir}/%{name}-%{version}/src/ipa/ipa-sign-install.sh %{_builddir}/%{name}-%{version}/%{_vpath_builddir}/src/ipa-priv-key.pem %{buildroot}/"@(InstallationPrefix)"/lib/libcamera/ipa_*.so \
 %{nil}
 
 %install
-- 
2.43.0

